docusaurus.config.js

```diff
@@ -94,6 +94,13 @@ export default async function createAsyncConfig() {
           contextualSearch: true,
           searchPagePath: '/search',
         },
+        announcementBar: {
+          id: "calico_ebpf",
+          content: "Help shape the future! 🚀 We're collecting feedback to improve Calico opensource, and your insights are key. Click <a href=\"https://forms.gle/HbRP4BoZQrVsrhHV8\">here </a> to learn more.",
```

Comment:
```suggestion
          content: "Help shape the future! 🚀 We're collecting feedback to improve Calico Open Source, and your insights are key. Click <a href=\"https://forms.gle/HbRP4BoZQrVsrhHV8\">here </a> to learn more.",
```

Reply:
LGTM, with one nit.

Two thoughts: 
* I know that I would be more likely to complete an 8-part survey if 1) I knew how long it would take and 2) it was clear that I was getting swag. You might consider clarifying those points at the start of the survey.
* Is the image/messaging at the start of the survey page up to date? I suppose container security is generic enough, but I had the idea that might be on the outs.

Let me know when this is ready and I can merge straightaway.